### DISCUSSION TITLE
How silent is airplane mode with DivestOS

### DISCUSSION THREAD

**Discussion by martin-braun** [Divested-Mobile/DivestOS-Build#221] [Q&A] [answered]
I read somewhere even airplane mode toggled on will not prevent the phone from connecting to a tower to leak the IMEI. Even without having a SIM inserted it shall connect to a tower. I think to enable emergency calls without a SIM, but I haven't tried to emergency call in airplane mode.

So if I want to avoid trigonometry tracking through network towers, having bluetooth and wifi scanning off, but the SIM still inserted (and sometimes used), is airplane mode secure to be a shadow in the dark with DivestOS? (Of course Wifi is a different attack vector that is a topic on its own, Orbot is probably a good choice here).

OnePlus 5, but curious about other devices as well.

**Comment by SkewedZeppelin** [comment, marked as answer]
so airplane mode is only set once the system boots, it is possible during the bootup process that the modem will briefly poll nearby towers before airplane mode is engaged.

in normal operation airplane mode will only be disengaged if the user explicitly tries to place an emergency call or uses the emergency panic button shortcut

regarding other location vectors please read this: https://divestos.org/misc/gnss.txt
and this: https://divestos.org/pages/network_connections

**Reply by martin-braun** [reply]
@SkewedZeppelin Very good news, thanks for your quick response. 👍
